h2o-py/tests/testdir_apis/H2O_Module/pyunit_h2oget_timezone_DEPRECATED.py

```diff
@@ -18,7 +18,7 @@ def h2oget_timezone():
 
     timezones = h2o.list_timezones()
     assert_is_type(timezones, H2OFrame)
-    assert timezones.nrow==460, "h2o.get_timezone() returns frame with wrong row number."
+    assert timezones.nrow==468, "h2o.get_timezone() returns frame with wrong row number."
```

Comment:
Ehh, how the change affects this test?

Reply:
I also upgraded JodaTime. The new version provides more timezones. Looks okay so far